Js/popup.js

```diff
@@ -21,7 +21,7 @@ const projectList = [
       bootstrap: 'Bootstrap',
       ruby: 'Ruby',
     },
-    projectImage: '../image/card1.png',
+    projectImage: '../image/card22.png',
```

Comment:
- [ ] Your images are broken because of the incorrect link. :sob: But don't worry if you would remove one dot from the start of your link then the image will be shown up :blush: Please do it with all cards images :+1:

| Your Link | Correct Link |
|-------|------|
|'../image/card22.png'|'./image/card22.png'|

Reply:
@HAMMAS-SALEEM Thank you. But his is my last review how can I do that?

Reply:
@HAMMAS-SALEEM  I don't really understand what you are asking me to change? I works fine for me when I make it **'../image/card22.png'**